README.md

```diff
@@ -1,7 +1,15 @@
 # TFExplorer
+[![Build Status](https://travis-ci.org/hardlyhuman/TFExplorer.svg?branch=master)](https://travis-ci.org/hardlyhuman/TFExplorer)
+[![alt text](https://patrolavia.github.io/telegram-badge/chat.png "Logo title")](https://t.me/joinchat/ME1g6xj6Y9vLcQhjiMSNyw)  
+  
+**TFExplorer** enables developers to quickly find out the relevant TensorFlow API's for their needs & requirements.
+It makes different Tensorflow APIs easily searchable.
+This is the repository for backend code.
```

Comment:
Break this line 7 in a new line

Reply:
Line 8 Also

Reply:
By leaving a blank line will also create a break for a line.